src/shims/unix/foreign_items.rs

```diff
@@ -60,6 +60,21 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 let result = this.open(args)?;
                 this.write_scalar(Scalar::from_i32(result), dest)?;
             }
+            "epoll_create1" => {
```

Comment:
Are these standard unix functions, or Linux-specific? If they are for Linux specifically, they should be in `linux/foreign_items.rs`.